File: C/Iniciante/1173 - Preenchimento de Vetor I..c
/*
* Autor: Pierre Vieira
* Data de submissão: 23/09/2018 15:21:39
*/
#include <stdio.h>
#define tamanho 10
int main(){
    int v[tamanho], x, numero;
    scanf("%d", &numero);
    for(x = 0; x <= tamanho; x++){
        if(x == 0){
            v[x] = numero;
        }
        else{
            v[x] = 2*v[x-1];
        }
    }
    for(x = 0; x <= tamanho-1; x++){
        printf("N[%d] = %d\n", x, v[x]);
    }
}

File: C#/Iniciante/1173 - Preenchimento de Vetor I.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 16/05/2020 22:43:28
*/
using System;

namespace URI {
    class Program {
        static void Main(string[] args) {
            int[] vet = new int[10];
            int number;
            number = int.Parse(Console.ReadLine());
            vet[0] = number;
            for (int i = 1; i < 10; i++) {
                vet[i] = vet[i - 1] * 2;
            }
            for (int i = 0; i < 10; i++) {
                Console.WriteLine($"N[{i}] = {vet[i]}");
            }
        }
    }
}
